src/components/LineChart/LineChartData.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+
+import {
+  Chart as ChartJS,
+  LineElement,
+  PointElement,
+  LinearScale,
+  CategoryScale,
+  Tooltip,
+  Legend,
+} from 'chart.js';
+import { Line } from 'react-chartjs-2';
+
+ChartJS.register(LineElement, PointElement, LinearScale, CategoryScale, Tooltip, Legend);
+
+const LineChartData = ({ activeTab }) => {
+  const labels = [
+    '2025-04-01', '2025-04-02', '2025-04-03', '2025-04-05', '2025-04-10',
+    '2025-04-12', '2025-04-15', '2025-04-20', '2025-04-28', '2025-04-30'
+  ];
+
+  const datasetsByTab = {
```

Comment:
your download, upload and ping data should have the same amount of ISPs cause the ISPs all return a download speed, upload speed and ping(latency) data